.bamboo/E2E-tests/execute.sh

```diff
@@ -7,7 +7,7 @@ if [ "$DB" = "mysql" ]; then
 elif [ "$DB" = "postgresql" ]; then
   COMPOSE_FILE="cypress-E2E-tests-postgres.yml"
 else
-  echo "Invalid database type. Please choose either mysql or postgresql."
+  echo "Invalid database type. Please choose either mysql or postgres."
```

Comment:
```suggestion
  echo "Invalid database type. Please choose either mysql or postgresql."
```
The if-condition above checks for `postgresql`.